Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -47,6 +47,7 @@ public static void UseOrganisationEndpoints(this WebApplication app)
                 await useCase.Execute(userUrn)
                     .AndThen(organisations => Results.Ok(organisations)))
             .Produces<List<Model.Organisation>>(200, "application/json")
+            .Produces(404)
```

Comment:
List organisation should never return 404

Reply:
Thats true 404 will not come from a list. I have removed it.